scalding-commons/src/main/java/com/twitter/scalding/commons/thrift/ThriftStreamCompare.java

```diff
@@ -1,13 +1,7 @@
 package com.twitter.scalding.commons.thrift;
```

Comment:
how old is this? make sure to grab the one from my internal branch, several fixes went in.